extension/src/index.ts

```diff
@@ -43,6 +46,8 @@ export async function activate(context: vscode.ExtensionContext) {
   activateIvyBrowser(context, '');
   context.subscriptions.push(YamlEditorProvider.register(context));
   setStatusBarMessage('Axon Ivy Extension activated');
+
+  const messenger = await deferredMessenger.promise;
   return messenger.diagnosticApi();
```

Comment:
this await prevents the possibility of creating a new project in an empty workspace. because 'process-editor.activate' is not triggered in an empty workspace until a new project is added.

Reply:
Good point, I changed it so we create the messenger instance once in the extension activate function. I hope that solves that issue.